docs/src/main/paradox/scala/http/release-notes.md

```diff
@@ -2,21 +2,115 @@
 
 ## 10.0.11
 
+See the [announcement](http://akka.io/news/2017/12/01/akka-http-10.0.11-released.html) and
+closed tickets on the [10.0.11 milestone](https://github.com/akka/akka-http/milestone/31?closed=1).
+
+This release adds the long awaited akka-http-caching module inspired by spray-caching.
+
+It also features a new implementation
+of the client pool infrastructure. This will allow us in the future to finally tackle many of the issues reported for the existing
+infrastructure like request timeouts, handling unread response entities, and other issues more easily.
+
+In an ongoing behind-the-scenes effort, [@jonas](https://github.com/jonas), [@jlprat](https://github.com/jlprat) and others
+continued to improve the structure of our documentation to consolidate Java and Scala documentation. This reduction in duplication
+of documentation content will allow us to make changes to the documentation more easily in the future. Thanks a lot!
+
+### New caching module, akka-http-caching
+
+In a several month long effort members from the community and the Akka team discussed and implemented the long-awaited replacement
+of spray-caching. The new module `akka-http-caching` got quite an overhaul over spray-caching and is now backed by
+[caffeine](https://github.com/ben-manes/caffeine).
```

Comment:
Maybe add something like this to clarify the use of `@ApiMayChange`.

```
The caching API may change based on feedback from real world usage
as well as improvements planned to make it into
[future releases](https://github.com/akka/akka-http/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3At%3Acaching+label%3A%22help+wanted%22).
As such we hope further collaboration within the community will
help us stabilize the API.
```

Reply:
Yep, good point.